.github/workflows/deploy-on-label.yml

```diff
@@ -61,8 +61,9 @@ jobs:
     - name: Circle CI Deployment Trigger
       id: curl-circle-ci
       if: steps.get-pr-state.outputs.STATE == 'success'
-      uses: Open-Source-Contrib/circle-ci-trigger-action@latest
+      uses: promiseofcake/circleci-trigger-action@v1
       with:
-        circle_ci_token: ${{ secrets.CIRCLE_CI_API_TOKEN }}
-        circle_ci_job: deploy-${{ steps.extract-deploy-env.outputs.DEPLOY_ENV }}
-        circle_ci_project_url: ${{ github.event.pull_request.head.ref }}
+        user-token: ${{ secrets.CIRCLE_CI_V2_API_TOKEN }}
```

Comment:
Ran into [this outstanding issue](https://discuss.circleci.com/t/v2-api-cannot-be-accessed-with-project-api-token/35914/2) with the new API in that it does not yet support `Project API Tokens`. The workaround applied here was adding my personal API token to the GitHub Secrets as `CIRCLE_CI_V2_API_TOKEN`, which is obviously not ideal. Once Circle CI has added support for Project API Tokens to the V2 API we will need to update this secret [here](https://github.com/raft-tech/TANF-app/settings/secrets/actions).

Reply:
Shall I just create a github-circleci service account for our use?

Reply:
That would work too, I think you'd have to create it in GH since Circle CI does auth through there.